s4e-web/cypress/page-objects/map/map-data-search.po.ts

```diff
@@ -0,0 +1,105 @@
+import { Core } from '../core.po';
+
+export class MapDataSearch extends Core {
+
+  static readonly pageObject = {
+    getGoToSearchDataBtn: () => cy.get('[data-e2e="data-sentinel-search"]'),
+    getSentinelsBtn: () => cy.get('[data-e2e="sentinel-checkbox"]'),
+    getSearchBtn: () => cy.get('[data-e2e="btn-search"]'),
+    getResultList: () => cy.get('[data-e2e="sentinel-search-result"]'),
+    getAllResultsSeparately: () => cy.get('[data-e2e="search-result-entry"]'),
+    getAllResultsDownloadBtn: () => cy.get('[data-e2e="download-link"]'),
+    getAllResultsDetailsBtn: () => cy.get('[data-e2e="show-details-button"]'),
+    getDetailsModal: () => cy.get('[data-e2e="modal-body"]'),
+    getResultDownloadAllArtifactsBtn: () => cy.get('[data-e2e="btn--download-all"]')
+  }
+
+  static goToSearchData() {
+    cy.server()
+    cy.route('GET', "/api/v1/config/sentinel-search").as("sentinelSearch")
+
+    MapDataSearch
+      .pageObject
+      .getGoToSearchDataBtn()
+      .click();
+
+    cy.wait('@sentinelSearch')
+
+    return MapDataSearch
+  }
+
+  static selectNthProductToSearch(number: number) {
+
+    cy.location('href').should('include', '/map/sentinel-search');
+
+    MapDataSearch
+      .pageObject
+      .getSentinelsBtn()
+      .eq(number)
+      .should("be.visible").and("exist")
+      .then((e) => {
+        Cypress.$(e).click();
+      })
```

Comment:
Why had you made such get around?

Reply:
If you're using some walk around's write why you're using it. The shorter and simpler description, the better.

Reply:
Without this test every time is failed because for cypress element is detached from DOM. 
![image](https://user-images.githubusercontent.com/59872096/103587245-1dd76780-4ee7-11eb-985b-02727a28b8fe.png)